src/command_modules/azure-cli-reservations/azure/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Use like other modules:
```
import pkg_resources
pkg_resources.declare_namespace(__name__)
```

Same with the other init files.

Reply:
fixed